packages/strapi-utils/lib/models.js

```diff
@@ -260,5 +260,59 @@ module.exports = {
 
   getVia: (attribute, association) => {
     return _.findKey(strapi.models[association.model || association.collection].attributes, {via: attribute});
+  },
+
+  convertParams: (entity, params) => {
+    if (!entity) {
+      throw new Error('You can\'t call the convert params method without passing the model\'s name as a first argument.');
+    }
+
+    const model = entity.toLowerCase();
+
+    if (!strapi.models.hasOwnProperty(model)) {
+      return this.log.error(`The model ${model} can't be found.`);
+    }
+
+    const connector = strapi.models[model].orm;
+
+    if (!connector) {
+      throw new Error(`Impossible to determine the use ORM for the model ${model}.`);
+    }
+
+    const convertor = strapi.hook[connector].load().getQueryParams;
```

Comment:
It means that every time a request hits the server, the hook is reloaded? If we have to, we should store the module itself in `strapi.hook.xx.module` and avoid to reload it on every request.